lib/src/config_util.cc

```diff
@@ -0,0 +1,29 @@
+#include <internal/config_util.hpp>
+#include <cctype>
+
+namespace hocon {
+
+    bool is_whitespace(char c) {
+        switch (c) {
+            case ' ':
+            case '\n':
+            // TODO: these complain about char overflow, what do.
```

Comment:
The original library makes the assumption that *any* character could come up. Do we need that kind of support for our use cases?